### PR TITLE
Fixing applyUpdates() deprecation issue.

### PR DESCRIPTION
**Pull request by gupta-ashutosh5**
Done the changes to resolve following deprecation issue: https://www.drupal.org/node/3034742 in SKUFieldsManager.

### CODE REVIEW

File: modules/acm_sku/src/Entity/SKU.php [hunk 1]
```diff
@@ -444,76 +443,12 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
...
-            break;
-        }
-
-        // Check if we don't have the field type defined yet.
-        if (empty($field)) {
-          throw new \RuntimeException('Field type not defined yet, please contact TA.');
-        }
-
-        // We want to allow field labels to be translatable.
-        // Since we try to do this dynamically, we need to use t() with
```
Comment (nikunjkotecha) [inline modules/acm_sku/src/Entity/SKU.php:500]: @gupta-ashutosh5 please copy this comment to new place

Comment (gupta-ashutosh5) [reply modules/acm_sku/src/Entity/SKU.php:500]: done

**Comment by nikunjkotecha** [comment]
@sdelbosc @miromichalicka this is good to review and merge, it is tested already in project

**Comment by miromichalicka** [comment]
@nikunjkotecha I can still see the same error 
```
a:4:{s:11:"deprecation";s:311:"EntityDefinitionUpdateManagerInterface::applyUpdates() is deprecated in 8.7.0 and will be removed before Drupal 9.0.0. Use \Drupal\Core\Entity\EntityDefinitionUpdateManagerInterface::getChangeList() and execute each entity type and field storage update manually instead. See https://www.drupal.org/node/3034742.";s:5:"class";s:51:"Drupal\Tests\acq_sku\Functional\CategoryManagerTest";s:6:"method";s:16:"testSyncCategory";s:15:"triggering_file";s:106:"/home/travis/build/acquia/orca-build/docroot/core/lib/Drupal/Core/Entity/EntityDefinitionUpdateManager.php";}
```

**Comment by nikunjkotecha** [comment]
we are removing that call in this PR, I'll look into travis result for the PR

**Comment by nikunjkotecha** [comment]
looks like deprecation code scan passed: https://travis-ci.org/acquia/acquia_commercemanager/jobs/586390478

issues are around code and scope of that is beyond this PR (may be some related to PR as well but for sure not everything is related to this PR, for instance:

```
FILE: ...emanager/modules/acm/tests/src/Unit/DatabaseSessionStoreTest.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 53 | ERROR | [x] Expected "object" but found "\stdClass" for @var
    |       |     tag in member variable comment
    |       |     (Drupal.Commenting.VariableComment.IncorrectVarType)
 60 | ERROR | [x] Expected "object" but found "\stdClass" for @var
    |       |     tag in member variable comment
    |       |     (Drupal.Commenting.VariableComment.IncorrectVarType)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

@miromichalicka @sdelbosc how do you want to approach this? Should we create a separate issue and fix all these first in separate PR or ok to try and fix all of them in this PR?

**Comment by miromichalicka** [comment]
@nikunjkotecha it's failing in https://travis-ci.org/acquia/acquia_commercemanager/jobs/586390474

**Comment by nikunjkotecha** [comment]
> @nikunjkotecha it's failing in https://travis-ci.org/acquia/acquia_commercemanager/jobs/586390474

Looks like it is used somewhere else too, looking into it

**Comment by nikunjkotecha** [comment]
It's used in https://github.com/acquia/acquia_commercemanager/blob/8.x-1.x/modules/acm_sku/tests/src/Functional/CategoryManagerTest.php too which is creating issue

@gupta-ashutosh5  can you please remove that and push? Let's see if we still get that error mentioned in comment or we are good to simply remove it

**Comment by gupta-ashutosh5** [comment]
ok I will work on this and raise a PR

**Comment by gupta-ashutosh5** [comment]
@nikunjkotecha done the change. Also, I would like to know if it needs to be removed from https://github.com/acquia/acquia_commercemanager/blob/e26ad5b4b67e99b1ab630704d5b25f2930a5beb5/modules/acm_sku/src/SKUFieldsManager.php#L145 as well

**Comment by nikunjkotecha** [comment]
@miromichalicka @sdelbosc 

Isolated tests with recommended package versions passing now. Also tests for CategoryManagerTest are passing.

**Pull request merged**